src/components/Login.vue

```diff
@@ -6,9 +6,7 @@
       <br />
       <input type="password" placeholder="Password" v-model="password" />
       <br />
-      <v-btn @click="login">
-        <v-icon>mdi-login</v-icon>Log in
-      </v-btn>
+      <v-btn @click="login"> <v-icon>mdi-login</v-icon>Log in </v-btn>
```

Comment:
It would be good to use Prettier or equivalent with formatOnSave so that you are consistent.  Or maybe this was it fixing an inconsistency.